### ISSUE TITLE
Investigate issues with non-Filecoin anchors

### ISSUE THREAD

**Issue by maciejwitowski** [consensus-shipyard/ipc#960]
From @jsoares 

ETHGlobal participants have reported issues when trying to deploy to Base. While this isn't a priority use case for us, this should work, so we need to understand if it's user error or bugs in the stack. Fortunately, this should be a fairly easy determination, and just requires deploying ourselves on base or an eth testnet.

Report: https://filecoinproject.slack.com/archives/C05J93G9G4F/p1710605403547479

tl;dr:
- Trying to start the child validator it comes back with an error can't find subnet
- ipc-cli subnet list --parent/r84532 returns nothing

**Comment by raulk** [comment]
This is not something we support, or plan to in the short term. We are focused on Filecoin as a root for now. We'll reopen if that changes at some point.
